tripgo.swagger.yaml

```diff
@@ -89,6 +89,223 @@ paths:
           schema:
             $ref: '#/definitions/RegionInfoResponse'
 
+  /operatorInfo.json:
+    post:
+      tags:
+        - Configuration
+      summary: List of TSPs for a particular region.
+      description:
+        Retrieves detailed information about covered transport service providers for a specified region.
+      parameters:
+        - name: input
+          in: body
+          schema:
+            type: object
+            description: Use `modes` to filter by tranport modes. Only results for the given modes will be retuned. When ´full´ paramater is false, it will retrieve a list of operator names for the given region and ´modes´ and ´onlyRealTime´ if any. You can also retrieve information for a particular operator using ´operator´, providing its name as it was given before as a list member in this very same endpoint. ´onlyRealTime´ parameter to be used only when ´full´ is set false. The use of filters like ´modes´, ´onlyRealTime´ and/or ´operator´ could lead to a empty response. In order to prevent this, make sure such combinatios are posible, using previously fetched information.
+            properties:
+              region:
+                type: string
+                description: Region name/code from `regions.json`.
+              modes:
+                type: array
+                items:
+                  type: string
+                description: Public transit modes (pt_pub_*) for which results should be returned. [Mode identifier](http://skedgo.github.io/tripgo-api/site/advanced/#mode-identifiers) strings like `pt_pub` or `pt_pub_subway`. In absence, all modes are included.
+              operator:
+                type: string
+                description: Operator name to retrieve information from. When ´full´ parameter is false, this field is ignored. THIS COULD BE AN ARRAY TOO.
```

Comment:
We don't have to use the GTFS ID, just some kind of ID that we can use that's persistent (or at least more persistent than names in GTFS's agency.txt). If there's an ID in GTFS, let's use that, otherwise some internal code would be good — could be based on the name, but, again, that can and does change.